lib/ansible/plugins/lookup/csvfile.py

```diff
@@ -135,7 +136,7 @@ def read_csv(self, filename, key, delimiter, encoding='utf-8', dflt=None, col=1,
 
             for row in creader:
                 if len(row) and row[keycol] == key:
-                    return row[int(col)]
+                    return row[col]
```

Comment:
once i merge my PR , this part won't be needed as set_option will do the casting instead